src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,66 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+    private boolean isDeleted;
```

Comment:
```suggestion
    private Boolean isDeleted;
```
Рекомендують використовувати обгортки класів замість примітивів